@navikt/core/react/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../../tsconfig.json",
   "compilerOptions": {
     "noImplicitAny": false,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "types": ["vitest/globals"]
```

Comment:
This fixes type errors when using test(), expect() etc, e.g. `Cannot find name 'expect'.`

Not sure why it wasn't needed before.